build-phar.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ ! -f box.phar ]; then
+    wget https://github.com/humbug/box/releases/download/3.0.0-beta.4/box.phar -O box.phar
+fi
+
+# lock PHP to minimum allowed version
+composer config platform.php 7.2.0
```

Comment:
Isn't this already the case?

Reply:
https://github.com/Roave/BackwardCompatibilityCheck/blob/95b5c0d5ac37ff7cfeac8368ec7f0b3009d6fd09/composer.json#L5